### PR TITLE
Occupation GUI

### PR DESCRIPTION
**Pull request by turretDive**
Finally fixed the GUI issue where the occupation of each contact doesn't show up on the code.

### CODE REVIEW

**Review by Jnwkm — COMMENTED** [review]
Conflict between the Userguide.md, think you should have removed the extra field to resolve this.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ Format: `help`
 
 Adds a person to uNivUSal.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL [t/TAG]…​`
+Format: `add o/OCCUPATION n/NAME p/PHONE_NUMBER e/EMAIL [t/TAG]…​`
```
Comment (NicholasPatrick) [inline docs/UserGuide.md:89]: > Conflict between the Userguide.md, think you should have removed the extra field to resolve this.

Yes, specifically, it's missing `a/ADDRESS` here.

**Review by NicholasPatrick — COMMENTED** [review]
Just change this to get rid of the conflict.

File: docs/UserGuide.md (hunk 1 above)
Comment (turretDive) [reply docs/UserGuide.md:89]: Oh I see it now, thanks for the advice!

**Pull request merged**